### PR TITLE
feat: added event remove - admin user

### PR DESCRIPTION
**Pull request by vivianamck**
## Issue ticket number and link

https://trello.com/c/M2oo0Xuz

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [X] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

- Button remove non-functional event

## What is the new behavior?

- Added functionality event remove
- Created remove event Endpoint
- Added Modal for answer 
- Added events list update

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/api/baseRequest.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default class baseRequest {
     return axios.put(this.getUrl(method), data)
   }
 
-  delete = (method, params) => {
+  eliminate = (method, params) => {
```
Comment (jlora-mck) [inline src/api/baseRequest.js:56]: Delete reads better than eliminate

File: src/components/EventsAdminView/EventAdminView.tsx [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
 
 import FullLayout from '../../hocs/FullLayout'
 import Headquarters from '../Headquarters/Headquarters';
 import DashboardFilters from '../Dashboard/DashboardFilters';
 import { Conference, ConferenceFilters, Headquarter } from '../../shared/entities';
 import { sortAscending, sortDescending } from '../../tools/sorting';
+import EventsApi from '../../api/events'
+
+const modalStyle = {
+  position: 'absolute' as 'absolute',
```
Comment (jlora-mck) [inline src/components/EventsAdminView/EventAdminView.tsx:14]: why we need to cast here ?

File: src/components/EventsAdminView/EventAdminView.tsx [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
```
Comment (jlora-mck) [inline src/components/EventsAdminView/EventAdminView.tsx:19]: is this correct ?

File: src/components/EventsAdminView/EventAdminView.tsx [hunk 4]
```diff
@@ -78,6 +109,26 @@ export default function EventAdminView({
...
+          setTitle('Successfully added!')
+          updateEvents(idEvent)
+        } else {
+          setTitle('emimination failed')
+        }
+      })
+    } catch (error) {
+      setTitle('Elimination failed')
+      console.error(error);
+    }
```
Comment (jlora-mck) [inline src/components/EventsAdminView/EventAdminView.tsx:127]: Refactor to use async/await

File: src/components/EventsAdminView/EventAdminView.tsx (hunk 3 above)
Comment (vivianamck) [reply src/components/EventsAdminView/EventAdminView.tsx:19]: Yes, use of Modal @mui/material documentation

File: src/components/EventsAdminView/EventAdminView.tsx (hunk 2 above)
Comment (vivianamck) [reply src/components/EventsAdminView/EventAdminView.tsx]: Done

File: src/components/EventsAdminView/EventAdminView.tsx (hunk 4 above)
Comment (vivianamck) [reply src/components/EventsAdminView/EventAdminView.tsx]: Done

**Review by Sofiag8 — APPROVED** [review]

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**